app.json

```diff
@@ -17,7 +17,7 @@
     },
     "JHIPSTER_REGISTRY_VERSION": {
       "description": "Version of the registry to deploy.",
-      "value": "7.0.0"
+      "value": "7.2.0"
```

Comment:
Ideally, the next registry version should be `7.1.0` to align with semantic versioning guidelines. I am not sure about the release process followed over here. IMO, we should decide next version during release as versioning should consider the kind of changes included in release (breaking or non-breaking)

Reply:
Honestly, I set that value because I thought that jhipster-registry had to share version with jhipster.  In the pom file, I saw that you set the version to "X.X.X-SNAPSHOT" while you are developing a new version, so I will change pom.xml and app.json to 7.1.0-SNAPSHOT. If you prefer to set it to other value, tell me and I will change it

Reply:
Look at the reference: https://github.com/jhipster/jhipster-registry/pull/491

This version is used for Heroku deployment and should match to the version of JHipster Release, not to generator-jhipster

Reply:
In #491, aren't we talking about `jhipster-registry` and not the `generator-jhipster`?

Reply:
btw, this comment was obsolete and I suggested changing to `7.1.0`